src/inspector_js_api.cc

```diff
@@ -237,6 +250,11 @@ void IsEnabled(const FunctionCallbackInfo<Value>& args) {
 
 void Open(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
+  if (!env->access_control()->
```

Comment:
The same checks are pretty repetitive. Why not introduce a macro, or a method on/that receives `Environment`, and would just check against it, instead of calling `access_control()->has_permission()` over and over?

f.e:

```cpp
#define THROW_IF_INSUFFICIENT_PERMISSIONS(env, permission)\
    if (!env->access_control()->has_permission(permission))\
        return THROW_ERR_ACCESS_DENIED(env);
```

and then, you can replace all those expressions with 
```cpp
THROW_IF_INSUFFICIENT_PERMISSIONS(env, AccessControl::accessInspectorBindings);
```

Reply:
Or a template - though personally I am fine with the explicitness of this the way it is now

Reply:
I don't have any problem with the explicitness.

Reply:
It's just a thought. I personally prefer less repetition of this style. It also makes it easier to remember when adding additional features that require this kind of permissions check, since you only have to remember this.

But I guess this is not something big enough to bikeshed over.
@benjamingr what do you mean by a template?

Reply:
@gioragutt Done in 41d6bccddc15a1439a721d038ebabe7500d7613e :)